wire/protocol.go

```diff
@@ -151,6 +151,12 @@ func (f ServiceFlag) String() string {
 	return s
 }
 
+const (
```

Comment:
nit: declare up with the service flags? This location seems a bit arbitrary

Reply:
Addressed